app/instance-initializers/violations-helper.js

```diff
@@ -0,0 +1,11 @@
+import ViolationsHelper from 'ember-a11y-testing/utils/violations-helper';
+
+export function initialize(application) {
+  window.violationsHelper = new ViolationsHelper();
```

Comment:
@Robdel12 instead of guarding all the places `violationsHelper` is mentioned, how do you feel about just excluding the initializer in production like: https://github.com/ember-a11y/ember-a11y-testing/blob/master/index.js#L55

Treat it like a public API in development/testing. No fiddling with ENV.